src/ngx_http_lua_cache.c

```diff
@@ -73,6 +80,7 @@ ngx_http_lua_cache_load_code(ngx_log_t *log, lua_State *L,
                       key, err);
         lua_pop(L, 2);
         return NGX_ERROR;
+#endif
```

Comment:
Better add a comment here after `#endif` to indicate what condition for this macro branch? This branch is a bit large.